src/runtime/model/animated.rs

```diff
@@ -376,7 +376,10 @@ impl ColorStops {
             let g = v0.get(j + 2)?.tween(v1.get(j + 2)?, t, &easing);
             let b = v0.get(j + 3)?.tween(v1.get(j + 3)?, t, &easing);
             let a = v0.get(j + 4)?.tween(v1.get(j + 4)?, t, &easing);
-            let stop = peniko::ColorStop::from((offset as f32, peniko::Color::rgba(r, g, b, a)));
+            let stop = peniko::ColorStop::from((
+                offset as f32,
+                peniko::Color::new([r as f32, g as f32, b as f32, a as f32]),
```

Comment:
And same question here...

Reply:
(But I do think this one is fine...)

Reply:
Normalized

https://docs.rs/peniko/0.2.0/peniko/struct.Color.html#method.rgba

Logic is correct